index.d.ts

```diff
@@ -873,7 +873,7 @@ declare namespace Eris {
       reason?: string
     ): Promise<Webhook>;
     executeWebhook(webhookID: string, token: string, options: WebhookPayload & { wait: true }): Promise<Message>;
-    executeWebhook(webhookID: string, token: string, options: WebhookPayload): Promise<void>;
+    executeWebhook(webhookID: string, token: string, options: WebhookPayload): Promise<any>;
```

Comment:
should this be `Promise<Message | any>`?
JSDocs look like that.

Reply:
No, it returns something different based on the wait parameter